listServer.js

```diff
@@ -72,16 +76,27 @@ function denyRequest (req, res) {
 // --- Functions --- //
 // API: Returns JSON list of servers.
 function apiGetServerList(req, res) {
-	if(req.method != "GET") {
+	if(req.method != "POST") {
```

Comment:
These chunks are probably not needed anymore but I'll clean them up

Reply:
Okay :)